src/map/script.cpp

```diff
@@ -19591,6 +19591,53 @@ BUILDIN_FUNC(setunitname)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC(setbodysize)
+{
+	struct block_list* bl = NULL;
+	struct view_data* vd;
+	TBL_MOB* md = NULL;
+	TBL_NPC* nd = NULL;
+
+	if(!script_rid2bl(2,bl))
+	{
+		script_pushconststr(st, "Unknown");
+		return SCRIPT_CMD_FAILURE;
+	}
```

Comment:
```suggestion
	block_list* bl;

	if(!script_rid2bl(2,bl))
	{
		script_pushconststr(st, "Unknown");
		return SCRIPT_CMD_FAILURE;
	}

	TBL_MOB* md;
	TBL_NPC* nd;
```

Reply:
Do we want to return "Unknown"? We don't return anything on success, so either we return 0/1 on fail/success or nothing at all. Should also document it in the script command doc if we decide on either.